streaming/base/converters/dataframe_to_mds.py

```diff
@@ -201,12 +160,12 @@ def write_mds(iterator: Iterable):
         else:
             raise RuntimeError('TaskContext.get() returns None')
 
-        if isinstance(mds_path, str):  # local
-            output = os.path.join(mds_path, f'{id}')
-            out_file_path = output
+        if mds_path[1] == '':  # only local
```

Comment:
maybe i'm missing something obvious but where does the `mds_path` variable come from?

Reply:
it's from the `mds_path` variable in the larger `dataframeToMDS` function right?

Reply:
Yes, it is from the external function.